jenkins/scripts/pulse_in_pipeline_scanning/utils/common.py

```
@@ -10,6 +10,11 @@ def load_json(path):
         return json.load(f)
 
 
+def license_needs_review(license, result):
```

Ruff (0.15.12)

[error] Function argument `license` is shadowing a Python builtin

(A002)